svelte-kit-scss/src/lib/components/IssueSearch/IssueSearchControls/IssueSearchControls.svelte

```diff
@@ -7,27 +7,46 @@
 
   export let sortFilters: NavigationFilterOption[];
   export let stateFilters: NavigationFilterOption[];
+  export let milestoneFilters: NavigationFilterOption[];
 
-  const handleFilterSelect = async (option: NavigationFilterOption): Promise<void> => {
-    if (option.active) {
+  const navigateHref = async (href: string): Promise<void> => {
+    await prefetch(href);
```

Comment:
I don't think there is any benefit at all to calling  `prefetch(href)` and then calling `goto(href)`.

At this point `goto(href)` would be fine.